docs/index.rst

```diff
@@ -94,6 +94,14 @@ install <https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs>
 
     resource/**
 
+.. toctree::
```

Comment:
I couldn't get sphinx to build without warning without adding utils to the docs, since asgi requires a class from here as one of it's inputs